package.json

```diff
@@ -2,13 +2,14 @@
   "name": "path-intersection",
   "version": "2.2.1",
   "description": "Computes the intersection between two SVG paths",
-  "main": "intersect.js",
+  "module": "intersect.js",
```

Comment:
Do we need this? According to the docs, `main` is the field name for entry point: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#main

Reply:
```suggestion
  "main": "intersect.js",
```

Reply:
Let's try this, as the project itself is marked as `type="module"`.

However, this library is meant to be used in the browser, and it is a common convention to expose ES modules through the `module` field. 

Primary goal is to make bundlers happy. Usage in Node is only a secondary concern.